azure-pipelines.yml

```diff
@@ -124,7 +124,15 @@ stages:
             $manifest.SelectSingleNode("/*[local-name()='ApplicationManifest']/*[local-name()='Principals']").RemoveAll()
             $manifest.Save("src\Maestro\MaestroApplication\ApplicationPackageRoot\ApplicationManifest.xml")
             git diff
-          displayName: Remove Service Fabric RunAsPolicy
+          displayName: Remove Service Fabric RunAsPolicy from MaestroApplication
+
+        - powershell: |
```

Comment:
? What this change doing?

Reply:
This is a piece from Alex. The Manifest defines a user and policy used for local debugging. This snippet removes this user when deploying as a service. See also the changes in ApplicationManifest.